core/pkg/sync/http/http_sync.go

```diff
@@ -47,7 +48,9 @@ func (hs *Sync) ReSync(ctx context.Context, dataSync chan<- sync.DataSync) error
 }
 
 func (hs *Sync) Init(_ context.Context) error {
-	// noop
+	if hs.BearerToken != "" {
+		hs.Logger.Info("Deprecation Alert: bearerToken option is deprecated, please use authHeader instead")
+	}
```

Comment:
This alert regarding `bearerToken` might be redundant since the behavior is already documented. Open to not adding it